cmd/buildah/push.go

```diff
@@ -68,6 +71,7 @@ func init() {
 
 	flags := pushCommand.Flags()
 	flags.SetInterspersed(false)
+	flags.BoolVar(&opts.all, "all", false, "add all of the lists' images if the images to add are lists")
```

Comment:
copy/paste error in the description of the option?

Reply:
That is how the option is described in manfest list.